src/services/geometry/richgeo/ActsGeo.cc

```diff
@@ -1,10 +1,38 @@
-// Copyright 2022, Christopher Dilks
+// Copyright (C) 2022, 2023, Christopher Dilks
 // Subject to the terms in the LICENSE file found in the top-level directory.
-//
-//
 
 #include "ActsGeo.h"
 
+// constructor
+richgeo::ActsGeo::ActsGeo(std::string detName_, dd4hep::Detector *det_, std::shared_ptr<spdlog::logger> log_)
+  : m_detName(detName_), m_det(det_), m_log(log_)
+{
+  // capitalize m_detName
+  std::transform(m_detName.begin(), m_detName.end(), m_detName.begin(), ::toupper);
+
+  // set `WithinRadiator`
+  if(m_detName=="DRICH") {
+    auto zmax             = m_det->constant<double>("DRICH_zmax")  / dd4hep::mm;
+    auto aerogelZpos      = m_det->constant<double>("DRICH_aerogel_zpos")      / dd4hep::mm;
+    auto aerogelThickness = m_det->constant<double>("DRICH_aerogel_thickness") / dd4hep::mm;
+    WithinRadiator[richgeo::kAerogel] = [this,aerogelZpos,aerogelThickness] (double x, double y, double z) {
```

Comment:
Does it make sense to change the capture of `this` to `const` here? Even make the lambda itself const?

Reply:
`this` was only captured here to access `m_log`, and actually I only ever used `WithinRadiator` in #619, and no longer need it, so it can be removed entirely.